tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_container_test.ts

```diff
@@ -174,7 +174,7 @@ describe('Graph Executions Container', () => {
     })
   );
 
-  it('clicking the tensor name buttons dispatches graphOpFocused', fakeAsync(() => {
+  it('clicking tensor name dispatches graphExecutionFocused', fakeAsync(() => {
```

Comment:
`dispatches graphExecutionFocused when tensor name is clicked`.

Reply:
Done.